satellite/client/client.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package client
```

Comment:
General package names are problematic since they can easily start colliding with each other.

Reply:
I'm not sure on how I should structure the package and name them